languages/elixir/exercises/concept/enum/.docs/after.md

```diff
@@ -0,0 +1,132 @@
+`Enum` is a very useful module that provides a set of algorithms for working with enumerables. It offers:
```

Comment:
Would it perhaps be useful to explain what an enumerable is, or do students already know that?

Reply:
I added the same sentence here as in introduction.md, but to the section that lists examples of enumerables.

Reply:
👍